phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexToolForNonTxGlobalIndexIT.java

```diff
@@ -952,20 +998,22 @@ public void testIndexToolForIncrementalVerify_viewIndex() throws Exception {
                             viewIndexName, null, 0, IndexTool.IndexVerifyType.ONLY, "-st", String.valueOf(t1),
                             "-et", String.valueOf(t3));
             verifyCounters(it, 2, 2);
-
+/*
             // job with update on only one row
             it =
                     IndexToolIT.runIndexTool(directApi, useSnapshot, schemaName, viewName,
                             viewIndexName, null, 0, IndexTool.IndexVerifyType.ONLY, "-st", String.valueOf(t3),
                             "-et", String.valueOf(t4));
             verifyCounters(it, 1, 1);
```

Comment:
This should return 0 scanned and 0 valid because the view has where clause on val6 which won't qualify for the delete marker that is there on HBase row. Thank you @abhishek-chouhan for clarifying this. 

That means in order to consider a deleted row for incremental verification on a view with where clause, its upsert should also fall into the same window. Otherwise, any such mismatches will be caught in full verification. 

FYI @kadirozde @priyankporwal

Reply:
That seems like incorrect behavior though, because that delete statement _does_ delete a row that's part of the view.

Reply:
Based on our previous conversation that led to the filing of PHOENIX-5989, I've just updated this with a reference to 5989 and kept it commented out.